build/Targets/VSL.Versions.targets

```diff
@@ -73,8 +73,8 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <CommonNuGetMoniker>beta2</CommonNuGetMoniker>
-    <VSSemanticVersion>15.3</VSSemanticVersion>    
+    <CommonNuGetMoniker>beta3</CommonNuGetMoniker>
```

Comment:
@srivatsn A few lines down we set the RoslynNuGetPerBuildVersion to `1.0.0-$(CommonNuGetMoniker)`. So if I go back to `beta1` we would "go back in time" so to speak.